governance/pyth_staking_sdk/src/pyth-staking-client.ts

```diff
@@ -372,6 +372,45 @@ export class PythStakingClient {
     return sendTransaction(instructions, this.connection, this.wallet);
   }
 
+  public async unstakeFromAllPublishers(
+    stakeAccountPositions: PublicKey,
+    positionStates: (PositionState.LOCKED | PositionState.LOCKING)[],
+  ) {
+    const [stakeAccountPositionsData, currentEpoch] = await Promise.all([
+      this.getStakeAccountPositions(stakeAccountPositions),
+      getCurrentEpoch(this.connection),
+    ]);
+
+    const instructions = await Promise.all(
```

Comment:
you need to reverse the positions, this is because when we delete a LOCKING positions, it gets swapped with the last position, so the order of positions changes